addons/mrp/report/mrp_report_mo_overview.py

```diff
@@ -162,6 +162,12 @@ def _get_operations_data(self, production, level=0, current_index=False):
             'details': operations,
         }
 
+    def _compute_expected_operation_cost(self, workorder):
```

Comment:
Could you add the two methods on workorder model and call them with `workorder._compute_expected_operation_cost()` ?